### DISCUSSION TITLE
[Feature] Sort shared albums or added setting/preference option?

### DISCUSSION THREAD

**Discussion by vulcanjedi** [immich-app/immich#3307] [Feature Request]
### The feature

Currently the timeline is defaulted and only sortable by descending order (new to old) but shared albums seem to be ascending (old to new)
I think this should be uniformly defaulted to descending (newest first ) across Immich or configurable via user settings?
Right now a shared album means user has to scroll down 40 yrs of pics for the newest ones!?

Past few releases have been massive steps and huge performance improvements....thank you!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #1689
